tests/test_app.py

```diff
@@ -12,10 +12,12 @@
 
 
 @pytest.mark.parametrize("config_file", CONFIG_FIXTURES)
-def test_app(config_file: pathlib.Path):
+@pytest.mark.parametrize("write_to_db", [True, False])
+def test_app(config_file: pathlib.Path, write_to_db: bool):
     runner = CliRunner()
     result = runner.invoke(
         main,
-        ["--config", str(config_file), "--date", "2022-1-1-11-50"],
+        ["--config", str(config_file), "--date", "2022-1-1-11-50"]
+        + (["--write-to-db"] if write_to_db else []),
     )
     assert result.exit_code == 0, traceback.print_exception(result.exception)
```

Comment:
would be good to check that thing have been written to the database in this test

Reply:
Yes I agree. I didn't do it out of lazyness 😅 . Will add.